academic-services/knowledge-exploration-service/tutorial-entity-linking.md

```diff
@@ -0,0 +1,116 @@
+
+## Set appropriate confidence score for linking
+
+Lastly, we'd have to determine an acceptable confidence score to link entities. Interpret API returns a log probability associated with each interpretation. This log probability can be seen as a "confidence score" for the given interpretation. Since the probability for an interpretation can range from 0 to 1, the log probability for an interpretation can be anywhere from negative infinity to 0.
+
+The interpretation log probability also included the log probability of the top matching entity. To get the pure interpretation log probability without considering the interpreted entity, we must subtract the entity log probability from the interpretation log probability.
+
+:::code language="powershell" source="linkPrivateLibraryData.ps1" id="snippet_interpret_log_probability_as_confidence_score":::
+
+The default MAKES grammar has various penalties associated with not matching paper titles directly. For example, a word matching a paper's fields of study instead of a title word will result a penalty of -1. The steepest penalty, -25 is not being able to match a word against any paper title or abstract. The sample entity linking script uses **-50** as the cut off confidence score for linking data. This allows for some penalization for entity linking, such as two title word mismatch.
+
+Depending on your linking strategy, you may have to set a different score. You may also want to design a custom grammar create a better scoring system for your scenario.
+
+## Choose MAKES entity attributes to include
+
+We start by determining which MAKES entity type we want to link our data against and what attributes can help enrich our private data. See the [MAKES Entity Schema](reference-makes-api-entity-schema.md) to explore the different types and entities and attributes available in MAKES' default index.
+
+In this tutorial, the sample private library publication records is minimal, with each record only containing a paper title and fake library URL. The goal of the tutorial is to augment the minimal library paper records with additional metadata from MAKES, enabling a robust semantic/keyword search experience.
+
+In the sample entity linking Powershell script, we will leverage paper entity's DOI, citation count, abstract, fields of study, authors, affiliations, journal, and conference information to enrich our data and support semantic/keyword search.
+
+We retrieve these information by constructing an Interpret request foreach link attempt with those attributes as the following:
+
+:::code language="powershell" source="linkPrivateLibraryData.ps1" id="snippet_interpret_request":::
+
+Once we're able to link a library paper against a MAKES paper entity successfully, we will then merge the entities together using the following helper function:
```

Comment:
>Once we're able to link a library paper against a MAKES paper entity successfully [](start = 0, length = 81)

If we're able to find an interpretation that meets our confidence cutoff criteria, ...